provider/aws/dist/rack.json

```diff
@@ -1094,7 +1100,18 @@
         }
       }
     },
-    "InstancesLifecycle": {
+    "InstancesLifecycleStarting": {
```

Comment:
Maybe `InstancesLifecycleLaunching` instead to match the official AWS naming convention?